blivet/udev.py

```diff
@@ -28,19 +28,25 @@
 from . import util
 from .size import Size
 from .flags import flags
+from .nvdimm import nvdimm
 
 import pyudev
 global_udev = pyudev.Context()
 
 import gi
 gi.require_version("BlockDev", "2.0")
+gi.require_version("GLib", "2.0")
 from gi.repository import BlockDev as blockdev
+from gi.repository import GLib
```

Comment:
As jenkins pointed out this isn't being used (nor is the `require_version` for it, above)